### PR TITLE
Fom confirmation

### PR DESCRIPTION
**Pull request by Vomcat**

### CODE REVIEW

**Review by p-szalast — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-PORT=3005
```
Comment (magnowak) [inline .env:1]: add newline

File: src/components/Cart/CartItem.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  id,
+  totalPrice,
+}) => {
+  const dispatch = useDispatch();
+
+  const addItemHandler = () => {
+    dispatch(cartActions.addItem({ name, quantity: 1, price, id, totalPrice }));
+  };
+
+  const removeItemHandler = () => {
```
Comment (magnowak) [inline src/components/Cart/CartItem.tsx:68]: what is the difference between removeItem and removeOneItem?

File: src/components/Cart/CartItem.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+  return (
+    <CartItemWrapper>
+      <p>{name}</p>
+      <ItemText>${totalPrice.toFixed(2)}</ItemText>
+      <ItemRemove onClick={removeOneItemHandler}>Remove</ItemRemove>
+      <ItemCount>
+        <ButtonPlus variant="grey" clickHandler={addItemHandler} />
+        <ItemCountNumber>{quantity}</ItemCountNumber>
+        <ButtonPlus
```
Comment (magnowak) [inline src/components/Cart/CartItem.tsx:84]: The name of the component is misleading, maybe better AddSubtractButton?

File: src/components/Cart/CartItem.tsx (hunk 2 above)
Comment (magnowak) [reply src/components/Cart/CartItem.tsx:68]: maybe removeItemEntry vs removeItemInstance?

File: src/components/Form/Form.tsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+  });
+
+  const onSubmit = (data: FormData) => {
+    setIsOpen(!isOpen);
+    setDataForm(data);
+  };
+
+  const onModalChangeHandler = () => {
+    setIsOpen(!isOpen);
+    console.log("testowe wywowalnie");
```
Comment (magnowak) [inline src/components/Form/Form.tsx:90]: remove

File: src/components/Form/FormConfimation.tsx [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+              {name} {lastName}
+            </p>
+            <p>{phoneNumber}</p>
+            <p>{email}</p>
+          </DataSummary>
+        </FormDataWrapper>
+        <OrderData>
+          <OrderSummary>
+            {orderItems.map((item) => (
+              <FormConfirmationItem
```
Comment (magnowak) [inline src/components/Form/FormConfimation.tsx:184]: would recommend destructuring item

File: src/hooks/useFetch.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+import { OrderItemProps } from "interfaces/Order";
+
+interface sendRequestProps {
+  url: string;
+  method?: string;
+  headers?: {};
+  body?: OrderItemProps | null;
+}
+
+const useFetch = () => {
```
Comment (magnowak) [inline src/hooks/useFetch.ts:11]: props for the custom hook

Comment (magnowak) [reply src/hooks/useFetch.ts:11]: As an extra, I recommend taking a look at RTK Query and rewriting the fetch logic using it

File: src/components/Form/FormConfimation.tsx (hunk 5 above)
Comment (Vomcat) [reply src/components/Form/FormConfimation.tsx:184]: Done

File: src/components/Form/Form.tsx (hunk 4 above)
Comment (Vomcat) [reply src/components/Form/Form.tsx:90]: Done

File: src/components/Cart/CartItem.tsx (hunk 2 above)
Comment (Vomcat) [reply src/components/Cart/CartItem.tsx:68]: Done

**Pull request merged**